taichi/transforms/compile_to_offloads.cpp

```diff
@@ -94,9 +94,6 @@ void compile_to_offloads(IRNode *ir,
   print("Simplified II");
   irpass::analysis::verify(ir);
 
-  irpass::constant_fold(ir);
-  print("Constant folded");
-
```

Comment:
Removing this cause it's already done in `Simplified II`?

Reply:
Yes.